README.md

```diff
@@ -50,7 +51,7 @@ Artifacts are publicly available on Maven Central starting from version *0.6*.
 Some or all of the following imports may be needed:
 ```scala
 import cats._
-import cats.implicits._
+import cats.syntax.all._
```

Comment:
Cats 2.2.0 has a new import for faster compilation times. It requires to remplace `cats.implicits._` with `cats.syntax.all._`:
https://github.com/typelevel/cats/releases/tag/v2.2.0